bin/svg-icon-generate.js

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env node
+const path = require('path');
+const argv = require('yargs')
+    .default('reactPureRender', true)
+    .default('radium', true)
+    .default('native', false)
+    .default('keepFillColor', false)
+    .default('template', path.join(__dirname, '..', 'template', 'icon.nunjucks'))
+    .usage('Usage: \n$0 --svgDir [./assets/svgs] --destination [./src/components/Icon.jsx]')
+    .demandOption(['svgDir','destination'])
+    .argv;
+const configureGenerator = require('../lib/configureGenerator');
+
+// const options = {
+//   svgDir: path.join(process.cwd(), 'icons'),
+//   destination: path.join(process.cwd(), 'components', 'Icon.js'),
+//   keepFillColor: false,
+//   native: false,
+//   radium: true,
+//   reactPureRender: true,
+//   template: path.join(__dirname, '..', 'template', 'icon.nunjucks'),
+// }
+
+console.log('Running SVG Icon generator with those options:');
```

Comment:
`Running SVG Icon generator with these options:` or even better `Running SVG Icon generator with the following options:`